README.md

```diff
@@ -164,3 +164,26 @@ git-upload-url: https://uploads.github.com/
 `cr` supports any format [Viper](https://github.com/spf13/viper) can read, i. e. JSON, TOML, YAML, HCL, and Java properties files.
 
 Notice that if no config file is specified, `cr.yaml` (or any of the supported formats) is loaded from the current directory, `$HOME/.cr`, or `/etc/cr`, in that order, if found.
+
+#### Notes for Github Enterprise Users
+
+For Github Enterprise, `chart-releaser` users need to set `git-base-url` and `git-upload-url` correctly, but the correct values are not always obvious to endusers.
+
+By default they are often along these lines:
+
+```
+https://example.com/api/v3/
+https://example.com/api/uploads/
```

Comment:
Maybe better use usual links for GitHub enterprise:

```suggestion
https://github.yourcompany.com/api/v3/
https://github.yourcompany.com/api/uploads/
```

Reply:
@pnowy I didn't apply this exactly as written (as I'd argue that there is very little guarantee about how companies name their GHE domains) but I did make a minor tweak to hopefully make the general point a bit more obvious.